### PR TITLE
convert to books

### PR DESCRIPTION
**Pull request by nathanallen**
- Switching to the "super-crud" endpoint and converting from "posts" back to "books".
- Added some clarifications in readme.
- Solution branch is now out of sync.

@tgaff please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Built on the top of the `$http` service, Angular’s `$resource` is a service th
 ```js
 angular.module('app', [..., 'ngResource']);
 ```
-1. In the application directory run python -m SimpleHTTPServer 8000.
+1. In the application directory run `budo --open`.
```
Comment (tgaff) [inline README.md]: budo?

Comment (tgaff) [inline README.md]: Here's a one-liner ruby server:
`ruby -rwebrick -e'WEBrick::HTTPServer.new(:Port => 3000, :DocumentRoot => Dir.pwd).start'`

Comment (nathanallen) [inline README.md]: They used it yesterday. Taking 24's lead here.
https://github.com/mattdesl/budo

File: README.md [hunk 2]
```diff
@@ -32,47 +32,47 @@ angular.module('app', [..., 'ngResource']);
...
-      $scope.post = Post.get({ id: 1843 }, function(data) {
+  app.controller('BooksController',function($scope, Book) {
+      $scope.book = Book.get({ id: 1843 }, function(data) {
         console.log(data);
-      }); // get() returns a single post
+      }); // get() returns a single book
 
-  $scope.posts = [];
-  $scope.newPost = {};
+  $scope.books = [];
```
Comment (tgaff) [inline README.md]: $scope?

**Comment by nathanallen** [comment]
closes #4

File: README.md [hunk 3]
```diff
@@ -93,6 +104,8 @@ angular.module('app', [..., 'ngResource']);
   });
   ```
 
+> Note: `{ id: "@_id"}` is a mapping between the placeholder in our route (e.g. `/books/:id)` and the name of the key that holds the id in the book object. And since databse is mongoDB our id is `_id`.
```
Comment (tgaff) [inline README.md]: databse -> the database

File: app.js [hunk 4]
```diff
@@ -1,17 +1,17 @@
-angular.module('postApp', ['ui.router', 'ngResource'])
-  .config( postConfig );
+angular.module('bookApp', ['ui.router', 'ngResource'])
+  .config( bookConfig );
 
-  function postConfig ( $locationProvider, $stateProvider, $urlRouterProvider ) {
-      // return to post-index if bad route request
+  function bookConfig ( $locationProvider, $stateProvider, $urlRouterProvider ) {
```
Comment (tgaff) [inline app.js]: `bookConfig.$inject = ['$locationProvider', '$stateProvider', '$urlRouterProvider']`

https://github.com/sf-wdi-labs/angular_style_guide#use-inject-vs-inline-annotation-for-dependency-injection

Comment (tgaff) [inline app.js]: could also rename to BookConfig but both of these are non-critical

Comment (nathanallen) [inline app.js]: done -- going to leave off injection until we deal with server-side.

**Pull request merged**